docs/.gitignore

```diff
@@ -0,0 +1 @@
+/diagrams/img/
```

Comment:
rimuovere questo gitignore. teniamo tutto nel gitignore alla radice (altrimenti per capire cosa è ignorato uno deve controllare in tutte le cartelle se esiste un altro gitignore)